enterprise/coderd/prebuilds/claim_test.go

```diff
@@ -325,21 +325,16 @@ func TestClaimPrebuild(t *testing.T) {
 
 			spy.claims.Store(0) // Reset counter because we need to check if any new claim requests happen.
 
-			wp, err := userClient.WorkspaceBuildParameters(ctx, userWorkspace.LatestBuild.ID)
-			require.NoError(t, err)
-
 			stopBuild, err := userClient.CreateWorkspaceBuild(ctx, workspace.ID, codersdk.CreateWorkspaceBuildRequest{
-				TemplateVersionID:   version.ID,
-				Transition:          codersdk.WorkspaceTransitionStop,
-				RichParameterValues: wp,
```

Comment:
why do we no longer need to pass in wp? is it because we include immutable rich params in our setup now?

Reply:
I'm not entirely sure since it was @evgeniy-scherbina who removed this, but from what I can see: this code seems unnecessary. The tests work with and without it.

It makes sense too since the previous build params are not passed when stopping a workspace (see `cli/stop.go`).

It _is_ required though for `start` operations, so I've restored that.

Well spotted 👍

Reply:
With initial fix - I had to delete it for Stop operation. Because otherwise it was considered mutating of immutable param.
With new fix in place - it should work either way.